src/Raven.Server/Documents/Handlers/Admin/Processors/Indexes/AdminIndexHandlerProcessorForDump.cs

```diff
@@ -31,6 +33,9 @@ protected override async ValueTask HandleCurrentNodeAsync()
                 return;
             }
 
+            if (index.SearchEngineType == SearchEngineType.Corax)
+                throw new NotSupportedInCoraxException("Index dump is not supported in Corax.");
```

Comment:
It should be thrown here:
https://github.com/ravendb/ravendb/blob/39c5635f0e8fcfffeed913a7929fd893231b9fa1/src/Raven.Server/Documents/Indexes/Index.cs#L5368-L5374

Reply:
Changed